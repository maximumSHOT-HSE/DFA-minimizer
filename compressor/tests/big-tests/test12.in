14 84 6
1 13 0
1 13 1
1 8 2
1 6 3
1 14 4
1 8 5
2 1 0
2 3 1
2 14 2
2 7 3
2 12 4
2 4 5
3 4 0
3 12 1
3 2 2
3 1 3
3 10 4
3 9 5
4 8 0
4 4 1
4 2 2
4 11 3
4 4 4
4 11 5
5 9 0
5 11 1
5 11 2
5 4 3
5 9 4
5 12 5
6 1 0
6 13 1
6 5 2
6 11 3
6 10 4
6 10 5
7 8 0
7 4 1
7 7 2
7 11 3
7 5 4
7 7 5
8 4 0
8 4 1
8 6 2
8 1 3
8 8 4
8 6 5
9 6 0
9 1 1
9 11 2
9 5 3
9 10 4
9 8 5
10 14 0
10 4 1
10 5 2
10 2 3
10 5 4
10 3 5
11 11 0
11 2 1
11 2 2
11 7 3
11 4 4
11 4 5
12 5 0
12 11 1
12 13 2
12 6 3
12 9 4
12 3 5
13 3 0
13 1 1
13 10 2
13 3 3
13 10 4
13 13 5
14 5 0
14 2 1
14 5 2
14 7 3
14 1 4
14 8 5
13
10
2 1 13 4 11 7 1 5 6 7 
