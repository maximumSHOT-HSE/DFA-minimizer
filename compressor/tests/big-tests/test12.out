14 84 6
2 1 0
6 1 0
9 1 1
13 1 1
3 1 3
8 1 3
14 1 4
11 2 1
14 2 1
3 2 2
4 2 2
11 2 2
10 2 3
13 3 0
2 3 1
13 3 3
10 3 5
12 3 5
3 4 0
8 4 0
4 4 1
7 4 1
8 4 1
10 4 1
5 4 3
4 4 4
11 4 4
2 4 5
11 4 5
12 5 0
14 5 0
6 5 2
10 5 2
14 5 2
9 5 3
7 5 4
10 5 4
9 6 0
8 6 2
1 6 3
12 6 3
8 6 5
7 7 2
2 7 3
11 7 3
14 7 3
7 7 5
4 8 0
7 8 0
1 8 2
8 8 4
1 8 5
9 8 5
14 8 5
5 9 0
5 9 4
12 9 4
3 9 5
13 10 2
3 10 4
6 10 4
9 10 4
13 10 4
6 10 5
11 11 0
5 11 1
12 11 1
5 11 2
9 11 2
4 11 3
6 11 3
7 11 3
4 11 5
3 12 1
2 12 4
5 12 5
1 13 0
1 13 1
6 13 1
12 13 2
13 13 5
10 14 0
2 14 2
1 14 4
13
8
1 2 4 5 6 7 11 13 
